packages/subversion/build.sh

```diff
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://subversion.apache.org
 TERMUX_PKG_DESCRIPTION="Centralized version control system characterized by its simplicity"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.14.3
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=1.14.4
 TERMUX_PKG_SRCURL=https://www.apache.org/dist/subversion/subversion-${TERMUX_PKG_VERSION}.tar.bz2
```

Comment:
```suggestion
TERMUX_PKG_SRCURL=https://downloads.apache.org/subversion/subversion-${TERMUX_PKG_VERSION}.tar.bz2
```
Also please turn on auto update

Reply:
No worries I will address that issue later. https://github.com/termux/termux-packages/actions/runs/11672829075/job/32502268199?pr=22110#step:4:41 indicates auto update querying works. You can update this package together or leave to auto update.

Reply:
Thankfully, for aarch64, x86 and x86_64, the auto update succeeded, but unfortunately, for 32-bit arm it failed, with the error "`curl: (28) Failed to connect to downloads.apache.org port 443 after 269423 ms: Couldn't connect to server`" in this log https://github.com/termux/termux-packages/actions/runs/11690137938/job/32554446631
That seems like it might have been an intermittent download failure, because I can't think of any real technical difference between how the source URL is downloaded for 32-bit vs 64-bit. I hope there is a way for that to eventually automatically retry that build.